source/content/guides/composer-convert.md

```diff
@@ -35,72 +30,82 @@ export site=yoursitename
 
   ```bash{promptUser:user}
-  git checkout -b composify
+  git remote add ic git@github.com:pantheon-upstreams/drupal-project.git && git fetch ic && git checkout -b composerify ic/master
   ```
 
-  You can replace `composify` with a branch name of your choosing, but all following steps assume this name.
-
-## Setup a Multidev (optional)
-If your Pantheon account has access to [multidev](/multidev), create a multidev to push your new code to:
-
-```bash{promptUser:user}
-git push origin composify && terminus env:create $site.dev composify
-```
+  You can replace `composerify` with another branch name. If you do, remember to adjust the other examples in this doc to match.
 
-This will setup the multidev environment to receive and demo our changed code.
+1. Copy your configuration from the default branch:
 
-## Create a New Composer Project
+  ```bash{promptUser:user}
+  git checkout master sites/default/config
+  git mv sites/default/config/* config
+  git commit -m "Pull in configuration from default branch"
+  ```
 
-1. In your local terminal, from the repository root of your Pantheon site, move a directory up:
+1. Compare your `pantheon.yml` with the new `pantheon.upstream.yml` and check for conflicts:
 
   ```bash{promptUser:user}
-  cd ..
+  git diff master:pantheon.yml pantheon.upstream.yml
   ```
 
-2. Use Composer to create a new project, using the [Pantheon Drupal 8 Composer](https://github.com/pantheon-systems/example-drops-8-composer) repository:
+1. If you need your `pantheon.yml`, copy it over (be sure to edit if needed):
 
-    ```bash{promptUser:user}
-    composer create-project pantheon-systems/example-drops-8-composer $site-composer
-    cd $site-composer
-    ```
+  ```bash{promptUser:user}
+  git checkout master pantheon.yml
+  git add pantheon.yml
+  git commit -m 'Copy my pantheon.yml'
+  ```
 
-This will create a new directory based on the example project [pantheon-systems/example-drops-8-composer](https://github.com/pantheon-systems/example-drops-8-composer) in the `$site-composer` directory.
+### (Optional) Return to Drupal 8
 
-### Copy pantheon.upstream.yml
-Since the drops-8 upstream has a `pantheon.upstream.yml` and the example-drops-8-composer upstream does not, copy over our old file for the platform to properly load the site. From the `$site-composer` directory, run:
+If you’re not ready to update to Drupal 9 yet. (Advisable: test the Composer conversion before testing the Drupal 9 upgrade)
+Note that you might want to hand-edit your composer.json and remove the drupal/core entry altogether. The empty drupal/core entry from the steps above will cause drupal/core to be erased and redownloaded frequently
 
 ```bash{promptUser:user}
-cp ../$site/pantheon.upstream.yml .
+cd upstream-configuration/
+composer remove zaporylie/composer-drupal-optimizations --no-update
+composer require drupal/core-recommended:^8 --no-update
+cd -
+composer config --json "extra.patches.drupal/core" '{}'
```

Comment:
```suggestion
```
No longer needed because the configuration this targets in the upstream was removed in https://github.com/pantheon-upstreams/drupal-project/commit/5ebb134bf1024320b805f4ac2a10d4a7e4908211